src/components/Header/Header.jsx

```diff
@@ -1,5 +1,22 @@
+import mainLogo from '../../assets/img/logo.png';
+import commonStyles from '../../styles.scss';
+import classes from './Header.module.scss';
```

Comment:
why classes and styles? what is the difference?

Reply:
Done. This line was removed because it didn't use in this file.